app/controllers/concerns/breadcrumbable.rb

```diff
@@ -2,6 +2,10 @@
 
 module Breadcrumbable
   def save_origin_page_for(trainee)
-    OriginPage.new(trainee, session, request).save
+    PageTracker.new(trainee.slug, session, request).save_as_origin!
```

Comment:
worth using `@page_tracker ||= PageTracker.new(trainee.slug, session, request)` so we can just refer to it as `page_tracker.save_as_origin!`?

Reply:
Good idea!